src/main/java/com/google/sps/servlets/ProfileServlet.java

```diff
@@ -96,7 +99,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       boolean glutenFree = Boolean.parseBoolean(request.getParameter("glutenFree"));
       boolean dairyFree = Boolean.parseBoolean(request.getParameter("dairyFree"));
       String[] allergies = request.getParameterValues("allergies");
-
+      
```

Comment:
nit: is this change required?

Reply:
The change from String[] to ArrayList<String> was necessary due to the way datastore returned the list and datastore does not store String[]. The type of id was changed from long to String because the String given by userService gave a numberFormatException when I tried to convert it to long.